monitoring/local_monitoring/common.sh

```diff
@@ -429,7 +429,10 @@ function kruize_local_demo_setup() {
 	echo -n "🔄 Installing kruize! Please wait..."
 	kruize_start_time=$(get_date)
 	if [ ${CLUSTER_TYPE} != "local" ]; then
-	  kruize_install &
+	  	docker pull quay.io/kruize/kruize_operator
+	  make bundle IMG="kruize/kruize-operator:v0.0.1" 
```

Comment:
I see a makefile with bundle target in the kruize-operator, is this the one being used here? If so, you should clone that repo too.